queries.sql

```diff
@@ -21,5 +21,65 @@ SELECT * FROM animals WHERE neutered = true;
 /*Find all animals not named Gabumon*/
 SELECT * FROM animals WHERE NOT name LIKE 'Gabumon';
 
-/*Find all animals with a weigh between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)*/
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+/*Find all animals with a weigh between 10.4kg and 17.3kg 
+(including the animals with the weights that equals precisely 10.4kg or 17.3kg)*/
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/*Inside a transaction UPDATE the animals table by SETting the species column to unspecified. Verify that change was made. 
+Then roll back the change and verify that species columns went back to the state before transaction.*/
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+/*Inside a transaction:
+Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
+Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+Commit the transaction.
+Verify that change was made and persists after commit.*/
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NLL;
```

Comment:
- [ ] Please be careful with the typos, here you are missing a `u` in the word `NULL`

Reply:
THanks for catching that, I did it correctly in the console but for some reason I wrote it like that in my code.